translations/bg/LC_MESSAGES/messages.po

```diff
@@ -910,13 +909,13 @@ msgstr "Highest quiz score"
 #, fuzzy
 msgid "highscore_explanation"
 msgstr ""
-"On this page you can look through programs created by other Hedy users. You "
-"can filter on both a Hedy level and adventure. Click on \"View program\" to "
-"open a program and run it. Programs with a red header contain a mistake. You "
-"can still open the program, but running it will result in an error. You can "
-"of course try to fix it! If the creator has a public profile you can click "
-"their username to visit their profile. There you will find all their shared "
-"programs and much more!"
+"On this page you can look through programs created by other Hedy users. "
+"You can filter on both a Hedy level and adventure. Click on \"View "
```

Comment:
Hi @jpelay!

Do we understand why this PR rewrites the layout of all po files (it might mess up Weblate?)

Reply:
Oh I didn't realize that when submitting the PR. I just copy-pasted the commands listed in the wiki, so I'm not sure why this happened. I'll try this again and fix it!

Reply:
I've been looking into this and it has to do with the width of the output lines. For some reason maybe this has changed? According to their docs the maximum output is 76, but if I pass that width in this branch, everything still changes. Maybe we can add a `--no-wrap` option or set the maximum amount of `width` ourselves.

Reply:
Ok! I'll do that tomorrow!

Reply:
So I have the answer as to why this keeps happening: the way Weblate and gettext calculate the length of the string is fundamentally different, one uses byte lenght and the other character length, there is an [issue for that](https://github.com/WeblateOrg/weblate/issues/6350) on the Weblate repository that is marked as wontfix, so no matter what width we choose, there will always be a difference in the way the lines are wrapped. 

The way I see we can fix this is by installing a plug-in to Weblate and set the option to not wrap lines ([apparently someone had problems](https://github.com/WeblateOrg/weblate/issues/6350) with that too, but I'm not sure if they messed up something or if its actually an issue, still they marked it as wontfix), but as it concerns to this particular PR I don't think there's a way to prevent the changes done to the line spacing, at least in the state that main is right now, perhaps after applying the plug-in and if the lines aren't wrapped I can apply the same option here and there won't be all of those differences.

What would like to do? Can I install the plug-in to Weblate or are there other considerations?

Reply:
Hi @jpelay! Great find about the plugin! I don't think this is a pressing issue, let's rather focus on the other open plans for now, but I will make an issue so we can pick it up in the future.